src/test/java/com/microsoft/sqlserver/jdbc/KerberosTest.java

```diff
@@ -75,6 +76,30 @@ public void testUseDefaultJaasConfigConnectionStringPropertyFalse() throws Excep
         }
     }
 
+    @Tag(Constants.Kerberos)
```

Comment:
I thought we changed this to `kerberos` with a lowercase k?

Reply:
nvm

Reply:
It's already lowercase, the value that is.

Reply:
Yeah I saw that in the other PR

Reply:
just add tag to whole class then we won't need to do it for each test and @BeforeAll won't need to be run